matrix/server.py

```diff
@@ -943,10 +943,12 @@ def room_get_messages(self, room_id):
         if not room_buffer.prev_batch:
             return False
 
+        lines = W.window_get_integer(W.current_window(), "win_chat_height")
+
         uuid, request = self.client.room_messages(
             room_id,
             room_buffer.prev_batch,
-            limit=10)
```

Comment:
Right this should have been max_initial_sync_events instead. We should use this instead and make your way a configurable option. Pretty sure not everyone wants their chat window filled.